### ISSUE TITLE
Dashboard improvement branch bugs

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#9701]
- [x] VoiceOver reads the tab header as presentation

- [x] Focus indicator not visible for focused tab header

- [x] Should be able to navigate between tabs using arrow keys

- [x] VoiceOver reads the tab header twice: home home 

- [x] insights not announcing  the loading status and loading spinner not showing when loading or refreshing happens

- [x] vertical tab and horizontal tab have different background color, shall we unify them?

- [x] database dashboard left tab is much less wide than the server dashboard's because it only has home tab

- [ ] dashboard doesn't get the focus when first launched

- [x] high contrast theme - part of the border for toolbar and left nav pane not visible - Kim
![Screen Shot 2020-03-24 at 5 30 14 PM](https://user-images.githubusercontent.com/13777222/77489455-6361f280-6df5-11ea-924d-8031efbac7a3.png)

- [x] high contrast theme - properties widget has two bottom borders - Kim
![image](https://user-images.githubusercontent.com/31145923/78615294-efc4da00-7825-11ea-8d1e-663b2d6534ba.png)

- [x] high contrast theme - style of active tab is not consistent - Kim
![Screen Shot 2020-03-25 at 11 01 37 AM](https://user-images.githubusercontent.com/13777222/77570079-78d42c80-6e88-11ea-9b6e-6430208a5867.png)

- [x] data virtualization extension
move the task to the home page toolbar
<img width="1378" alt="Screen Shot 2020-03-25 at 3 28 36 PM" src="https://user-images.githubusercontent.com/13777222/77591533-60c2d400-6ead-11ea-9b0a-be6164218912.png">

- [x] increase minimum size for collapsible left nav
![image](https://user-images.githubusercontent.com/40371649/77601017-cec7c500-6ec6-11ea-90e3-695d57777f26.png)

- [x] rectify Refresh action on toolbar and refresh in "...". I didn't see anything happen when I clicked either of them.

- [x] ensure dashboard widget documentation is up to date (accessible by the Learn More action)

- [x] navigating from Home to another tab results in Database Size widget being blank
![image](https://user-images.githubusercontent.com/40371649/77601315-a68c9600-6ec7-11ea-957b-59fd293116a1.png)

- [x] Make databases list a full content pane experience

- [x] sql agent operators tab: horizontal scroll bar almost invisible when screen width is decreased:
![image](https://user-images.githubusercontent.com/40371649/77601518-36cadb00-6ec8-11ea-87df-3179c7166c85.png)
[alan] existing issue.

- [x] tab header has line-height of 45px, which is much higher than the mockups:
![image](https://user-images.githubusercontent.com/40371649/77602045-a8efef80-6ec9-11ea-8e62-eb7168256ee2.png)
vs.
![image](https://user-images.githubusercontent.com/40371649/77602066-b3aa8480-6ec9-11ea-9eb2-6e73b21ff737.png)

**Breadcrumbs**

- [x] Text styling - Kim

- [x] Link styling - Kim

- [x] Breadcrumb arrow size - Kim

- [x] Spacing -horizontal padding - Kim

**Left Navigation Bar**

- [x] Tab text styling - Kim

- [x]  Tab category styling - Kim

- [x] Collapse chevron icon (left) - should be smaller with padding to match the height of the toolbar

- [x] Tab alignment - should align with where toolbar ends. - Alan

- [x] Spacing - horizontal padding . Icons should be positioned 24px from the Connections viewlet and be aligned with the breadcrumb.  - Kim

**Server/Database properties**

- [x] Remove title "SERVER DASHBOARD" because it isn't in mockups - Kim

- [ ] Property layout. Should be listed in 2 columns instead of each property/value pair in its own column

- [x] Property name text style - Kim

- [x] Property value text style - Kim

- [x] Remove ellipses because refresh action has been moved to the toolbar - Kim

- [x] Move collapse chevron icon to bottom center of widget instead of current top right location - Kim

**Widget Styling**

- [x] Title text styling - Kim

- [x] Last updated text styling - Kim

- [x] Widget padding to match mockup

- [x] handle multiple contributed tasks widget , for example machine learning tab. - alan
